docs/current_docs/manuals/user/remotes/remote-repositories.mdx

```diff
@@ -11,3 +11,91 @@ Here is an example of a Go builder Dagger Function that accepts a remote Git add
+- **Monorepo support**: Append `/subpath` to access a specific subdirectory within the repository.
+- **Version specification**: Add `@version` to target a particular version of the module. This can be a tag, branch name, or full commit hash. If omitted, the default branch is used.
+
+These options are optional.
+
+### SSH Refs and Authentication
+
+Here is an overview of how Dagger handles authentication when using SSH refs:
+
+Dagger mounts the socket specified by your host's `SSH_AUTH_SOCK` environment variable to the engine. This is essential for SSH refs, as most Git servers use your SSH key for authentication and tracking purposes, even when cloning public repositories. Ensure that the `SSH_AUTH_SOCK` environment variable is properly set in your environment when using SSH refs.
+
+For detailed instructions on setting up SSH authentication, including how to generate SSH keys, start the SSH agent, and add your keys, please refer to the following external documentation:
+
+[Connecting to GitHub with SSH](https://docs.github.com/en/authentication/connecting-to-github-with-ssh)
+
+This guide provides comprehensive information on:
+
+- Generating SSH key pairs
+- Setting up public key authentication on Linux, macOS, and Windows
+- Configuring SSH clients
+- Best practices for SSH key management
+
+You can ensure that your SSH authentication is properly configured for use with Dagger by following this guide.
+
+### Best Practices
+
+For quick and easy referencing:
+- Copy the repository ref from your preferred Git server's UI.
+- Add a `/subpath` if you need to target a specific directory within the repository.
+- Append `@version` if you want to specify a particular version or commit.
+
+### Known Limitations and Workarounds
+
+This section outlines current limitations and provides workarounds for common issues. We're actively working on improvements for these areas.
+
+#### 1. Platform Support
+
+Currently, SSH refs are fully supported on UNIX-based systems (Linux and macOS). Windows support is under development. Track progress and contribute to the discussion in our [GitHub issue for Windows support](https://github.com/dagger/dagger/issues/8313).
+
+#### 2. Multiple SSH Keys
+
+**Issue**: SSH forwarding may fail when multiple keys are loaded in your SSH agent.
+
+**Status**: Under [active investigation](https://github.com/dagger/dagger/issues/8288).
+
+**Workaround**: 
+1. Clear all loaded keys: `ssh-add -D`
+2. Add only the required key: `ssh-add /path/to/key`
+
+#### 3. Environment Variable Expansion
+
+**Issue**: Shell expansion characters (e.g., `~`) in the `SSH_AUTH_SOCK` environment variable are not automatically expanded.
+
+**Status**: Under [active investigation](https://github.com/dagger/dagger/issues/8337).
+
+**Workaround**: Use the full path instead of `~` when setting `SSH_AUTH_SOCK`.
+
+
+#### 4. CI/CD Integration (GitHub Actions)
```

Comment:
This does not seem to be either an issue or a limitation. WDYT of moving this into the Integration -> GitHub page instead?

Reply:
Yes, SGTM!